docs/src/modules/java/pages/quickstart/cr-value-entity-kafka-java.adoc

```diff
@@ -0,0 +1,160 @@
+= Quickstart: Customer Registry with Kafka in Java
+
+include::ROOT:partial$include.adoc[]
+include::java:partial$attributes.adoc[]
+
+
+Create a customer registry that includes publishing to Kafka. Package it into a container, and run it on Akka Serverless.
+
+== In this Quickstart you will learn:
+
+* How to add additional functionality, allowing to publish customer's events to Kafka.
+* How to package the customer registry into a container.
+* How to deploy and run the customer registry on Akka Serverless.
+
+== Before you begin
+
+* If you're new to Akka Serverless, {console}[create an account, window="console"] so you can try out Akka Serverless for free.
+* You'll also need to install the https://developer.lightbend.com/docs/akka-serverless/akkasls/install-akkasls.html[Akka Serverless CLI, window="new-doc"] if you want to deploy from a terminal window.
+* For this quickstart, you'll also need
+** https://docs.docker.com/engine/install[Docker {minimum_docker_version} or higher, window="new"]
+** Java {java-version} or higher
+** https://maven.apache.org/download.cgi[Maven 3.x or higher, window="new"]
+** https://github.com/fullstorydev/grpcurl#installation[`grpcurl`, window="new"]
+
+[NOTE]
+====
+If you want to bypass writing code and jump straight to the deployment:
+
+. Download the source code using the Akka Serverless CLI: 
+`akkasls quickstart download customer-registry-views-java`
```

Comment:
I think `customer-registry-views-java` should be `value-entity-kafka-java`?

Reply:
done